### PR TITLE
feat: setup baseline for backend

### PR DESCRIPTION
**Pull request by cedricgautier**
## Done
- Added elysia.js for backend of our app
- Backend application is going to run on port 3001

### CODE REVIEW

**Review by Gougoh — APPROVED** [review]

File: back/.gitignore
Comment (BarnabePILLIAUDIN) [inline back/.gitignore:1]: Is it necessary ?

File: back/README.md
Comment (BarnabePILLIAUDIN) [inline back/README.md:1]: Same as above

**Review by BarnabePILLIAUDIN — CHANGES_REQUESTED** [review]
Are the files above necessary ?

File: back/.gitignore
Comment (cedricgautier) [reply back/.gitignore:1]: Removed

File: back/README.md
Comment (cedricgautier) [reply back/README.md:1]: Removed

**Review by BarnabePILLIAUDIN — APPROVED** [review]
LGTM

**Review by Gougoh — APPROVED** [review]

**Pull request merged**